watermark/watermark.py

```diff
@@ -80,174 +78,159 @@ def watermark(self, line):
         IPython magic function to print date/time stamps
         and various system information.
         """
-        self.out = ''
+        output = []
         args = parse_argstring(self.watermark, line)
 
         if not any(vars(args).values()) or args.iso8601:
-            try:
-                dt = datetime.datetime.fromtimestamp(int(time()),
-                                                     datetime.timezone.utc)
-                iso_dt = dt.astimezone().isoformat()
-            except AttributeError:  # timezone only supported by Py >=3.2:
-                iso_dt = strftime('%Y-%m-%dT%H:%M:%S')
+            iso_dt = self._get_datetime()
 
         if not any(vars(args).values()):
-            self.out += iso_dt
-            self._get_pyversions()
-            self._get_sysinfo()
-
+            output.append({"Datetime": iso_dt})
```

Comment:
Can you change this to `"Last executed"`, `"Executed"` or similar? `"Datetime"` doesn't mean anything and isn't English.

Reply:
Hi James, I changed that to 'Last updated' to be compatible with the other one.

I hope that solves the problem.

Reply:
Great. Thanks.